.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,42 @@
+name: Bug Report
+description: Report any errors or incorrect behaviors to help improve PyCytominer.
```

Comment:
```suggestion
description: Report any errors or incorrect behaviors to help improve pycytominer.
```

pycytominer is lower case :)

Reply:
Whoops! This always messes me up since the logo has capitals. I will fix this for all of templates.